panda/src/cocoadisplay/cocoaGraphicsWindow.mm

```diff
@@ -1116,24 +1140,32 @@
   int refresh_rate;
   mode = CGDisplayCopyDisplayMode(_display);
 
-  // First check if the current mode is adequate.
-  if (CGDisplayModeGetWidth(mode) == width &&
-      CGDisplayModeGetHeight(mode) == height) {
-    return mode;
-  }
-
```

Comment:
It's just to make the code cleaner, it's just a shortcut to not loop over all the display mode, I can keep it but I have to add the new test there too

Reply:
I think the early bail-out was just paranoia on my end so that it wouldn't change display modes if some other, unknown property were to be different.  Maybe it's not very important.